for_developers/cli_guide.md

```diff
@@ -313,3 +313,35 @@ overrides:
       init_args:
         patience: 3
 ```
+
+## How to use OTX Workspace
+
+If we run a typical Training example, we'll have a folder like the one below as output.
+
+```console
+otx-workspace/
+    .latest/                       # As of the last training, we have a .latest with path information for config and checkpoint.
```

Comment:
Is there any reason we should hide the latest config?

Reply:
I didn't get it your comment, .latest is relative to the most recently trained work_dir.

Reply:
Why should be the directory name .latest, not latest. User can see .latest if they just type ls

Reply:
This is to omit some arguments in the CLI. I didn't think there would be many examples for users to use latest folder.
I didn't think about showing it at the same level as the other `work_dirs`.
What do you think, do you think it should be exposed via ls?

Reply:
I'm not sure, but let's leave it as it is